grr/server/grr_response_server/gui/ui/lib/api_translation/primitive.ts

```diff
@@ -85,7 +84,7 @@ export function decodeBase64(encodedString?: string): Uint8Array {
   try {
     decodedString = atob(encodedString);
   } catch (err) {
-    return new Uint8Array();
+    throw new Error('Invalid base64 encoding.');
```

Comment:
`err` might contain important information for developers. This code swallows `err` and raises a new Error with no information to debug this. Instead, I suggest removing the try-catch block to let the original error bubble up.